### PR TITLE
XW-4153 | Remove bower

### PR DESCRIPTION
**Pull request by hakubo**
https://wikia-inc.atlassian.net/browse/XW-4153

### CODE REVIEW

**Comment by hakubo** [comment]
For reason unknown to me yet, I had to change selectors in selenium tests
https://github.com/Wikia/selenium-tests/pull/2098

**Review by rybmat — APPROVED** [review]

File: lib/include-node-modules/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	},
+
+	included(app) {
+		this._super.included(app);
+
+		// import libraries into vendor.js that was merged with the vendor trees.
+		// In browser the library will be eval'd and run
+		// In fastboot, the library will not be eval'd
+		// Files below are concatenated to assets/vendor.js
+		app.import('vendor/jquery.cookie.js');
```
Comment (xkxd) [inline lib/include-node-modules/index.js:47]: i'd do a `[<arr-of-strings>].forEach((source) => app.import(source))` here

File: lib/include-node-modules/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	 * @param {Broccoli} tree
+	 */
+	treeForVendor(tree) {
+		let trees = [];
+
+		if (tree) {
+			trees.push(tree);
+		}
+
+		trees.push(this.getTree('visit-source/dist/visit-source.js', 'visit-source.js'));
```
Comment (xkxd) [inline lib/include-node-modules/index.js:30]: might also be
```
trees.push(
    this.getTree(),
    this.getTree(),
    etc...
)
```

**Review by xkxd — APPROVED** [review]

File: lib/include-node-modules/index.js (hunk 1 above)
Comment (hakubo) [reply lib/include-node-modules/index.js:47]: I like the idea, I'll implement it outside this PR

**Pull request merged**